src/bonus/BonusPriceTag.tsx

```diff
@@ -7,26 +7,32 @@ import {BonusProgramTexts, useTranslation} from '@atb/translations';
 import {View} from 'react-native';
 
 type Props = ComponentProps<typeof View> & {
-  price: number;
+  amount: number;
 };
-export const BonusPriceTag = ({price, ...props}: Props) => {
+export const BonusPriceTag = ({amount, ...props}: Props) => {
   const styles = useStyles();
   const {t} = useTranslation();
 
   return (
     <View {...props} style={[styles.container, props.style]}>
-      <ThemeText>{price}</ThemeText>
+      <ThemeText
+        accessible={true}
+        accessibilityLabel={t(BonusProgramTexts.costA11yLabel(amount))}
+      >
+        {amount}
+      </ThemeText>
```

Comment:
There is something iffy about the accessibility stuff here. I think we should either do it correctly, or not at all (if it is only used in map). We can have a talk about it.